test/specs/rules/require-var-decl-first.js

```diff
@@ -0,0 +1,94 @@
+var Checker = require('../../../lib/checker');
+var assert = require('assert');
+
+describe('rules/require-var-decl-first', function() {
+    describe('boolean', function() {
+        var checker;
+        beforeEach(function() {
+            checker = new Checker();
+            checker.registerDefaultRules();
+        });
+
+        describe('configure', function() {
+            it('should not report an error for value "true"', function() {
+                assert.doesNotThrow(function() {
+                    checker.configure({ requireVarDeclFirst: true });
+                });
+            });
+
+            it('should report an error for value "false"', function() {
+                assert.throws(function() {
+                    checker.configure({ requireVarDeclFirst: false });
+                });
+            });
+        });
+
+        describe('rule checks', function() {
+            var scenarios = [
```

Comment:
Would be easier to have a list of valid snippets. We can assume that they'd have an errorCount of zero if they were members.